polkadot/runtime/parachains/src/paras_inherent/benchmarking.rs

```diff
@@ -26,6 +26,20 @@ use polkadot_primitives::v7::GroupIndex;
 use crate::builder::BenchBuilder;
 
 benchmarks! {
+	enter_empty {
+		let scenario = BenchBuilder::<T>::new()
+			.build();
+
+		let mut benchmark = scenario.data.clone();
+
+		benchmark.bitfields.clear();
+		benchmark.backed_candidates.clear();
+		benchmark.disputes.clear();
```

Comment:
Are these really needed if we just called `build()` on a default `BenchBuilder` instance?